index.js

```diff
@@ -115,3 +64,108 @@ app.on('activate', () => {
         createWindow();
     }
 });
+
+function getArgs() {
+    // Get the parameters
+    let failed = false;
+    var argv = yargs(hideBin(process.argv))
+        .help("h").alias("h", "help")
+        .usage("Usage: $0 [<options>]")
+        .example("$0 -b zepto.bin", "Start the emulator and execute zepto.bin in the emulator")
+        .option('rom', {
+            type: 'string',
+            alias: 'r',
+            description: 'Select a prebuild romdisk or a local image',
+            nargs: 1,
+        })
+        // TODO: 
+        // .option('binary', {
```

Comment:
This implies having an OS  image already loaded into the system or having the user specify one in pair with this argument.